lib/util.js

```diff
@@ -86,7 +86,8 @@ const inspectDefaultOptions = Object.seal({
   showProxy: false,
   maxArrayLength: 100,
   breakLength: 60,
-  compact: true
+  compact: true,
+  clockTime: 1000
```

Comment:
I think you misspelled this... you're using `inspectDefaultOptions.maxClockTime` further down at any rate.